launch_ros/doc/source/actions.rst

```diff
@@ -0,0 +1,50 @@
+`launch_ros` actions
+====================
+
+Action frontend arguments names
+-------------------------------
+
+Most of the `launch_ros` Python actions (:class:`launch_ros.actions`) are `available through the XML and YAML frontends <https://docs.ros.org/en/rolling/How-To-Guides/Launch-file-different-formats.html>`_.
+The name of the action arguments might vary between the underlying Python implementation and the frontends.
+
+:class:`launch_ros.actions.Node`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Action frontend name: ``node``.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Python argument name
+     - frontend argument name
+   * - ``arguments``
+     - ``args``
+   * - ``executable``
+     - ``exec``
+   * - ``exec_name``
+     - ``exec_name``
+   * - ``namespace``
+     - ``namespace``
+   * - ``node_name``
+     - ``node-name``
+   * - ``package``
+     - ``pkg``
+   * - ``parameters``
+     - ``param`` (``name``, ``value``)
+   * - ``remappings``
+     - ``remap`` (``from``, ``to``)
```

Comment:
For arguments like this, it's a bit more complicated than just providing these names. What people really need are examples. I'm tempted to link to each action's frontend tests (at least for the ones that have frontend tests in `test_launch_ros`), like this one for the `Node` action: https://github.com/ros2/launch_ros/blob/38152b2d04c6b6f126d31726ac9e12888b2e2e1f/test_launch_ros/test/test_launch_ros/frontend/test_node_frontend.py. However, I'm not sure how to do it properly (i.e., how to get a distro-specific link to the source code, or maybe the test file itself can be included in the docs?), and this could get outdated, so I'd just overall like to get some feedback.